Source/Core/CoplanarPolygonGeometry.js

```diff
@@ -468,12 +521,24 @@ CoplanarPolygonGeometry.createGeometry = function (polygonGeometry) {
 
   const results = PolygonGeometryLibrary.polygonsFromHierarchy(
     polygonHierarchy,
+    defined(textureCoordinates),
     projectPoints,
     false
   );
   const hierarchy = results.hierarchy;
   const polygons = results.polygons;
 
+  const textureCoordinatePolygons = textureCoordinates
+    ? PolygonGeometryLibrary.polygonsFromHierarchy(
+        textureCoordinates,
+        true,
+        function (identity) {
```

Comment:
This function should be declared as a variable, then passed in.

Reply:
Fixed

Reply:
Fixed.